es8/2018-01/jan-24.md

```diff
@@ -64,7 +64,7 @@ JHD: We have tests, editor-reviewed spec PR. Asking for Stage 4. Any objections?
 (Gabriel Isenberg)
 
  - [proposal](https://github.com/tc39/proposal-optional-chaining)
- - [slides]()
```

Comment:
Instead of deleting these, please replace them with the actual links, which are on the [agenda](https://github.com/tc39/agendas/blob/master/2018/01.md). https://docs.google.com/presentation/d/1tAxG8y-lfMty2-qdiCAtQjD9rS8xzBe9Y4b_ti9DIMQ/edit?usp=sharing